tests/unit/progressbar/progressbar_core.js

```diff
@@ -1,28 +1,29 @@
 module( "progressbar: core" );
 
-test( "markup structure", function() {
-	expect( 5 );
-	var element = $( "#progressbar" ).progressbar();
-	ok( element.hasClass( "ui-progressbar" ), "main element is .ui-progressbar" );
-	ok( !element.hasClass( "ui-progressbar-indeterminate" ),
-		"main element is not .ui-progressbar-indeterminate" );
-	equal( element.children().length, 1, "main element contains one child" );
-	ok( element.children().eq( 0 ).hasClass( "ui-progressbar-value" ),
-		"child is .ui-progressbar-value" );
+test( "markup structure", function( assert ) {
+	expect( 6 );
+	var element = $( "#progressbar" ).progressbar(),
+		value = element.children().eq( 0 );
+
+	assert.hasClasses( element, "ui-progressbar ui-widget ui-widget-content" );
+	assert.lacksClasses( value, "ui-progressbar-complete" );
+
+	element.progressbar( "option", "value", 100 );
+	assert.lacksClasses( element, "ui-progressbar-indeterminate" );
```

Comment:
Why is this check after changing the value instead of upfront with the other checks?